docs/dataset.md

```diff
@@ -48,6 +48,8 @@ python ./tools/convert_datasets/otb100/download_otb100.py -o ./data/otb100/zips
 python ./tools/convert_datasets/vot/download_vot.py --dataset vot2018 --save_path ./data/vot2018/data
 ```
 
+- For GOT10k, there are 1k videos in official training set that are forbidden from "using to train models then testing on VOT" (as required by [VOT Challenge](https://www.votchallenge.net/vot2019/participation.html)). Following the common setting, we use the video IDs on [got10k_vot_train_split.txt](https://github.com/visionml/pytracking/blob/master/ltr/data_specs/got10k_vot_train_split.txt) to train and [got10k_vot_val_split.txt](https://github.com/visionml/pytracking/blob/master/ltr/data_specs/got10k_vot_val_split.txt) to evaluation. You can download these split txts to `$MMTRACKING/data/got10k/full_data/train_data`.
```

Comment:
add comments at the last to explain that the video ids in `vot_train` and `vot_val` are both from the train set of GOT10K